streaming/src/main/scala/io/gearpump/streaming/task/TaskActor.scala

```diff
@@ -285,10 +303,14 @@ class TaskActor(
       doHandleMessage()
   }
 
-  def minClock: TimeStamp = {
-    Math.max(life.birth, Math.min(upstreamMinClock, minClockAtCurrentTask))
-  }
+  /**
+   * @return min clock of this task
+   */
+  def getMinClock: TimeStamp = thisMinClock
```

Comment:
Why not use def minClock: TimeStamp = _minClock

It is confusing to see thisMinClock